packages/babel-polyfill/scripts/build-dist.sh

```diff
@@ -1,17 +1,14 @@
 #!/bin/sh
 set -ex
 
-BROWSERIFY_CMD="../../node_modules/browserify/bin/cmd.js"
```

Comment:
Do not have any assumption on node_modules structures and invoke via `yarn` instead.